feature-libs/product-configurator/rulebased/components/attribute/types/radio-button/configurator-attribute-radio-button.component.html

```diff
@@ -67,11 +67,22 @@
           >{{ value.valueDisplay }}</label
         >
       </div>
+
       <div class="cx-value-price">
         <cx-configurator-price
           [formula]="extractValuePriceFormulaParameters(value)"
         ></cx-configurator-price>
       </div>
     </div>
+    <div *ngIf="attribute.uiType === uiType.RADIOBUTTON_ADDITIONAL_INPUT">
```

Comment:
could also be a getter _hasAdditionalInput_

Reply:
@Uli-Tiger:  Yes agree, then we don't need to define uiType on component level which I find a bit confusing (had copied that)

Reply:
And that getter (or those getters) could directly access the validationType and decide which component to include

Reply:
Here I would suggest to check `validationType` to verify whether an input value is `numeric` or `alphanumeric`

Reply:
To check `UI type` is not necessary at this place because it has already been done in the form

Reply:
Yes, agree to that. We will have a controller method that takes care